pennylane_lightning/lightning_kokkos/lightning_kokkos.py

```diff
@@ -467,24 +304,30 @@ def __init__(  # pylint: disable=too-many-arguments
             raise ImportError(
                 "Pre-compiled binaries for lightning.kokkos are not available. "
                 "To manually compile from source, follow the instructions at "
-                "https://pennylane-lightning.readthedocs.io/en/latest/installation.html."
+                "https://docs.pennylane.ai/projects/lightning/en/stable/dev/installation.html."
             )
 
-        super().__init__(wires=wires, shots=shots)
+        if batch_obs:
+            raise DeviceError(
+                "Lightning Kokkos does not support batch observables option. 'batch_obs' should be False"
+            )
```

Comment:
@vincentmr What do you think about making explicit the limitation of `batch_obs` for LK? Also, I think this will help the user

Reply:
I would not change the behaviour in this PR, but I don't mind either way. Let's see if anyone else had an opinion.